app/controllers/katello/api/registry/registry_proxies_controller.rb

```diff
@@ -12,7 +12,9 @@ class Api::Registry::RegistryProxiesController < Api::V2::ApiController
     before_action :authorize_repository_write, only: [:push_manifest]
     skip_before_action :check_content_type, only: [:start_upload_blob, :upload_blob, :finish_upload_blob,
                                                    :chunk_upload_blob, :push_manifest]
-    skip_after_action :log_response_body, :only => [:pull_blob]
+    if defined?(log_response_body)
```

Comment:
why check if its defined?

Reply:
When you run `skip_after_action` Rails doesn't define the method.. it was causing the web app not to load because `skip_after_action` had already been run on the Api::BaseController from which this inherits.

Reply:
After confirming this fixes the issue I'll probably just not run `skip_after_action` here, instead of checking

Reply:
Still not sure why @chris1984 and Jenkins are seeing this issue, but I didn't see the error locally